alibi/confidence/model_linearity.py

```diff
@@ -65,14 +65,12 @@ def _calculate_global_linearity(predict_fn: Callable, input_shape: Tuple, X_samp
     elif model_type == 'regressor':
         outs = predict_fn(X_samples)
         outs_shape = outs.shape[1:]
-        if len(outs.shape) == 1:
-            outs = outs.reshape(ss + (1,))  # shape=(nb_instances, nb_samples, 1)
-        else:  # if regression on multiple targets
-            outs = outs.reshape(ss + outs_shape)  # shape=(nb_instances, nb_samples, nb_targets)
+        outs = outs.reshape(ss + (1,)) if len(outs.shape) == 1 else outs.reshape(ss + outs_shape)
+
     else:
         raise ValueError("Passed 'model_type' not supported. Supported model types: 'classifier', 'regressor'")
     t_f = time() - t_0
-    logger.debug('predict time {}'.format(t_f))
+    logger.debug(f'predict time {t_f}')
```

Comment:
Historically we have the convention of not using f-strings with `logger`. See [here](https://klichx.dev/2019/11/09/f-strings-and-python-logging/).

Reply:
I believe the logging statements should be written with string interpolation `%s` syntax.